src/main/thrift/content/v1.thrift

```diff
@@ -290,7 +326,91 @@ struct ImageElementFields {
 
     12: optional string imageType
 
+    /** This field was added accidentally and will never be set. 
+      * isMaster only makes sense on an asset, not on an element 
+      */
     13: optional bool isMaster
```

Comment:
You might be safe to remove this - not sure if anyone is using the thrift formats yet?